tests/common/config_reload.py

```diff
@@ -21,7 +21,18 @@ def config_system_checks_passed(duthost, delayed_services=[]):
         logging.info("Checking failure reason")
         fail_reason = duthost.shell("systemctl list-units --state=failed", module_ignore_errors=True)
         logging.info(fail_reason['stdout_lines'])
-        return False
+        # For kvm testbed
+        # Service watchdog-control loads failed because of the expected Error `No module named 'sonic_platform'.
+        # Service system-health loads failed because of the failure of the above service.
+        # So in kvm testbed, if these two services load fail, we consider the system is healthy.
+        if duthost.facts["asic_type"] == "vs":
+            if any("watchdog-control.service" in service for service in fail_reason['stdout_lines']) \
+                    and any("system-health.service" in service for service in fail_reason['stdout_lines']):
+                pass
```

Comment:
This logic does not make sense to me. If all services are failed, it would still be considered as `pass`.